src/components/common/Navbars/NavBar.js

```diff
@@ -41,7 +41,7 @@ function NavBar(props) {
 
   useEffect(() => {
     if (role_id === 5) setBgColor('#9FB222');
-    else if (role_id < 5) setBgColor('#21C5B5');
+    else if (role_id < 5 && role_id > 2) setBgColor('#21C5B5');
```

Comment:
This was done so that only the admin's pages' headers would be changed to the set bgColor.